docs/examples/promise/06_make_transfer/index.js

```diff
@@ -24,13 +24,14 @@ async function main () {
   // here we split it out for the sake of readability
   const transfer = api.tx.balances.transfer(BOB_ADDR, 12345);
 
-  // Sign the transaction using our account
-  transfer.sign(alice, aliceNonce);
-
-  // Send the transaction and retrieve the resulting Hash
-  const hash = await transfer.send();
-
-  console.log(`transfer 12345 to Bob with hash ${hash}`);
+  // Sign and send the transaction using our account. Send the transaction and retrieve the resulting Hash
```

Comment:
Comment for the file renames above - don't include the .md files, CI will generate them

Reply:
Aren't the `.md` files what's actually needed? I thought only the `.html` files are being generated.

Reply:
Sorry, I meant promise/06_make_transfer/index.html above... but yes, just remove the generated files, here included due to the rename.

Reply:
Ok, I removed it :)